force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -5941,6 +6045,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Page-level error message about missing required fields</shortDescription>
         <value>These required fields must be completed: {0}</value>
     </labels>
+    <labels>
+        <fullName>commonNoneSpecific</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>None Specific</shortDescription>
+        <value>None Specific</value>
```

Comment:
Can confirm with Sam, but I think this should be "None Specified", or "Not Entered"? Something like that.